integration/tests/epochs/suite.go

```diff
@@ -441,7 +465,7 @@ func (s *Suite) SetApprovedNodesScript(ctx context.Context, env templates.Enviro
 		SetProposalKey(s.client.SDKServiceAddress(), 0, s.client.Account().Keys[0].SequenceNumber).
```

Comment:
To make the naming consistent, we could rename this method from `SetApprovedNodesScript` to `SubmitSetApprovedListTx`

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/b8764d41aa39b48d9971e01f862b0c20d3804d86